src/panels/OriginPanel/AnchorAimView/AnchorAimListEntry.tsx

```diff
@@ -24,10 +24,11 @@ export function AnchorAimListEntry({
   isCurrentAnchor,
   isCurrentAim,
   onSelectAnchor,
-  onSelectAim
+  onSelectAim,
+  ...props
```

Comment:
```suggestion
  ...styleProps
```
?  To make it clearer what these props contain